thunder/core/transform_common.py

```diff
@@ -467,16 +476,15 @@ def functionalize_inplace_ops(
     For example, :func:`thunder.torch.add_` is represented as a :class:`thunder.core.symbol.BoundSymbol`
     whose `subsymbols` are :func:`thunder.torch.add` and :func:`thunder.core.prims.copy_`. This function
     replaces it with a :class:`~thunder.core.symbol.BoundSymbol` of :func:`~thunder.torch.add`.
+
+    .. note::
+
+        Currently if an arg of ``computation_trace`` is passed to multiple in-place ops,
```

Comment:
I wonder how hard it would be to make this an error instead of a note...

Reply:
I bet you've noticed this one but #723 is trying to support such cases :)

Reply:
Right. :)